pom.xml

```diff
@@ -37,6 +60,12 @@
             <artifactId>couchbase-transactions</artifactId>
             <version>1.1.8</version>
         </dependency>
+        <dependency>
+            <groupId>io.reactivex</groupId>
+            <artifactId>rxjava-reactive-streams</artifactId>
+            <version>${reactivex.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why do we need RX Java?

Reply:
(resolved in Slack)
it provides WebTestClient used in tests